src/main/scala/com/redislabs/provider/redis/redisFunctions.scala

```diff
@@ -34,6 +34,22 @@ class RedisContext(@transient val sc: SparkContext) extends Serializable {
     new RedisKeysRDD(sc, redisConfig, readWriteConfig, keyPattern, partitionNum, null)
   }
 
+  /**
+    * @param keysOrKeyPattern an array of keys or a key pattern
+    * @param partitionNum number of partitions
+    * @return RedisHashRDD of related Key-Values stored in redis server
+    */
+  def fromRedisHLL[T](keysOrKeyPattern: T,
+                      partitionNum: Int = 3)
+                     (implicit redisConfig: RedisConfig = new RedisConfig(new RedisEndpoint(sc.getConf))):
```

Comment:
Please include a ReadWriteConfig parameter like in another methods. That would allow to override it if necessary for individual functions and not globally.